lib/emarsys.rb

```diff
@@ -48,7 +48,7 @@ class << self
     #
     # @return [String] domain which should be used to query the API
     def api_endpoint
-      @api_endpoint ||= 'https://suite5.emarsys.net/api/v2'
+      @api_endpoint ||= 'https://api.emarsys.net/api/v2/'
```

Comment:
If I got it right, the additional slash at the end causes the test failures on Travis.

Note that the specific path qualifier and the base endpoint are connected by `[…, …].join('/')` [in here the request class](https://github.com/Absolventa/emarsys-rb/blob/master/lib/emarsys/request.rb#L38))